index.html

```diff
@@ -48,7 +48,7 @@ <h3>Challenger 2</h3>
             </fieldset>
             <fieldset class="guess-field">
               <label for="guess2">Guess</label>
-              <input id="challenger2-guess" type="text" name="guess2">
+              <input id="challenger2-guess" type="number" name="guess2" min="1" max="100">
```

Comment:
I think this is a cool way to do some additional error handling and prevent players from inputting a number outside of that range.

Reply:
MDN has a great article that I just read this morning that talks about using the browser to handle errors as much as possible as its more reliable and faster than using the server side error handling. Article here: https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation